tests/optimizer/fairseq/test_fairseq_optimizer.py

```diff
@@ -64,11 +66,15 @@ def setUp(self):
         'lenpen': 2.0,
         'max_len_b': 140,
         'min_len': 55,
-        'no_repeat_ngram_size': 3
-    })
+        'no_repeat_ngram_size': 3,
+        #To test with el attention, please set 
+        #env variable USE_EL_ATTN=1
+        'use_el_attn':False, 
```

Comment:
Add a test for use_el_attn is True, and make sure it actually enabled.

Reply:
synched offline

Reply:
What's the conclusion here?

Reply:
User will need to set this env variable before running unit test with EL attn.

Reply:
Then `config.USE_EL_ATTN` can be used here to avoid the above parameter `use_el_attn:False`, which needs to be manually changed in the above case.

Also update [here](https://github.com/microsoft/fastseq/blob/main/tests/run_fastseq_tests.sh#L6) by adding `USE_EL_ATTN=1 python ....` to trigger the tests for EL-Attention?

BTW, CI seems to be hung for a while.

Reply:
EL attention is not supported for ProphetNet yet. Running all tests with USE_EL_ATTN=1 breaks regression.